lib/__tests__/standalone-cache.test.js

```diff
@@ -0,0 +1,73 @@
+"use strict"
+
+const path = require("path")
+const sinon = require("sinon")
+const standalone = require("../standalone")
+const FileCache = require("../utils/FileCache")
+const fixturesPath = path.join(__dirname, "fixtures")
+
+describe("standalone cache is enabled", () => {
+  let mockedFileCache
```

Comment:
Why are we mocking the file cache instead of testing the real thing? Couldn't we actually *use* the file cache in our tests, cleaning up after as needed, and then bypass some of the efforts below to test its internals?